Expensify/App
# Add new helpsite article for our new Budgets feature

### Details
Creating a new article for the Budgets feature we are launching next week!

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/337951

### Tests
1. Go to the new helpsite and head to Workspaces and Domain Settings
2. Make sure there is a new article called Budgets
3. Make sure formatting shows as expected

### Offline tests
N/A

### QA Steps

### PR Author Checklist

### Screenshots/Videos
No screenshots or videos yet, we'll add them later if necessary
